### ISSUE TITLE
Pytorch GroupNorm Support

### ISSUE THREAD

**Issue by bluesky314** [quic/aimet#415]
Am in need of some urgent help, am trying to do channel pruning on a model with torch.nn.GroupNorm but get 
`2021-01-27 16:36:29,878 - Winnow - ERROR - Unsupported op_type GroupNorm, dotted Layer.encoder.bn1, input_ops: [convolution_4]`

The error comes from `_set_default_input_output_masks` which raise the `NotImplementedError` as `op_connectivity` of the module is not any of those listed in the if-else clauses.

GroupNorm does not have any paramaters and should be easy to process if BatchNorm can be done. Is there can workaround or solution for this issue? Can we set it `op_connectivity` to something so we do not run in this error?

**Comment by quic-klhsieh** [comment]
Hi @bluesky314 ,

The mappings for op types to connectivities exist in a class which you can import and add to. It looks like you are working with Pytorch. Can you try adding these lines of code prior to the call to prune channels:

from aimet_common.winnow.winnow_utils import OpConnectivity, ConnectivityType
OpConnectivity.pytorch_dict[‘GroupNorm’]: ConnectivityType.direct

Please let us know if this solution works.

**Comment by bluesky314** [comment]
Thanks this issue is fixed
